docs/jwst/associations/association_reference.rst

```diff
@@ -263,36 +277,37 @@ Implementing ``dump()`` and ``load()``
 The base ``Association`` class defines the
 :meth:`~jwst.associations.association.Association.dump` and
 :meth:`~jwst.associations.association.Association.load` methods to
-serialize the data structure pointing to by the ``data`` attribute. If
+serialize the data structure pointed to by the ``data`` attribute. If
 the new rule uses the ``data`` attribute for storing the association
 information, no further overriding of these methods is necessary.
 
 However, if the new rule does not define ``data``, then these methods
 will need be overridden.
 
 Rule Registration
-=================
+-----------------
```

Comment:
This is so this section does not mistakenly get listed as a new page from `associations/index.rst` toctree.